bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableSession.java

```diff
@@ -120,21 +121,16 @@
     performWarmup();
   }
 
-  private static SslContextBuilder createGrpcSslBuilder() {
-    SslContextBuilder sslBuilder = GrpcSslContexts.forClient();
-    sslBuilder.ciphers(null);
-    return sslBuilder;
-  }
-
-  private static synchronized SslContextBuilder getSslBuilder() {
+  private synchronized static SslContext createSslContext() throws SSLException {
     if (sslBuilder == null) {
-      sslBuilder = createGrpcSslBuilder();
+      sslBuilder = GrpcSslContexts.forClient().ciphers(null);
+      if (OpenSsl.isAvailable()) {
+        LOG.info("gRPC is using the OpenSSL provider (tcnactive jar)");
```

Comment:
There's a request to have a parameter of some sort to use a programatic configuration of openSSL vs. boot jar (#506).  I think per connection is better for that case.  If you're OK with it, let's keep it here pending the fix for #506